### ISSUE TITLE
RC Test: wave-interference/2.0.0-rc.1 and waves-intro/1.0.0-rc.3

### ISSUE THREAD

**Issue by samreid** [phetsims/QA#405]
@ariel-phet @KatieWoe @arouinfar @kathy-phet, wave-interference/2.0.0-rc.1 and waves-intro/1.0.0-rc.3 are ready for RC testing. We would like this to be published before the beginning of school. Document issues in https://github.com/phetsims/wave-interference/issues and link to this issue.

Assigning to @ariel-phet for prioritization.

<details>
<summary><b>General RC Test</b></summary>

<h3>What to Test</h3>

- Click every single button.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
- If there is sound, make sure it works.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.
- Test all query parameters on all platforms. (See [QA Book](https://github.com/phetsims/QA/blob/master/doc/qa-book.md) for a list of query parameters.)
- Download HTML on Chrome and iOS.
- Make sure the iFrame version of the simulation is working as intended on all platforms.
- Make sure the XHTML version of the simulation is working as intended on all platforms.
- Complete the test matrix.
- Don't forget to make sure the sim works with Legends of Learning.
- Check [this](https://docs.google.com/spreadsheets/d/1umIAmhn89WN1nzcHKhYJcv-n3Oj6ps1wITc-CjWYytE/edit#gid=0) LoL spreadsheet and notify AR or AM if it not there.
- This is the first RC (even though one is marked rc.3), so please run a memory test.  
- When making an issue, check to see if it was in a previously published version
- If there is a console available, check for errors and include them in the Problem Description.

<h3>Focus and Special Instructions</h3>

Please be aware of prior dev testing which completed here: https://github.com/phetsims/QA/issues/389
Issues to make sure credits are correct are https://github.com/phetsims/wave-interference/issues/428 and
https://github.com/phetsims/waves-intro/issues/9

Note that build failures bumped the version number of waves-intro to rc.3 even though it is the first RC to be tested.

<h3>Link(s)</h3>

Wave Interference
- **[Simulation](https://phet-dev.colorado.edu/html/wave-interference/2.0.0-rc.1/phet/wave-interference_all_phet.html)**
- **[iFrame](https://phet-dev.colorado.edu/html/wave-interference/2.0.0-rc.1/phet/wave-interference_all_iframe_phet.html)**
- **[XHTML](https://phet-dev.colorado.edu/html/wave-interference/2.0.0-rc.1/phet/xhtml/wave-interference_all.xhtml)**
- **[Test Matrix](https://docs.google.com/spreadsheets/d/1wgAKZmWknpMONzv6sW5KcrfdTkB2q8Q0GCIGwDGUIUs/edit#gid=1529069804)**
- **[Legends of Learning Harness](https://developers.legendsoflearning.com/public-harness/index.html?startPayload=%7B%22languageCode%22%3A%22en%22%7D)**

<hr>

Waves Intro
- **[Simulation](https://phet-dev.colorado.edu/html/waves-intro/1.0.0-rc.3/phet/waves-intro_all_phet.html)**
- **[iFrame](https://phet-dev.colorado.edu/html/waves-intro/1.0.0-rc.3/phet/waves-intro_all_iframe_phet.html)**
- **[XHTML](https://phet-dev.colorado.edu/html/waves-intro/1.0.0-rc.3/phet/xhtml/waves-intro_all.xhtml)**
- **[Test Matrix](https://docs.google.com/spreadsheets/d/1C-tenj9bMA_fVjWR56KcQCNT44xNlVxjf4I0xWtTCl4/edit#gid=1374469194)**
- **[Legends of Learning Harness](https://developers.legendsoflearning.com/public-harness/index.html?startPayload=%7B%22languageCode%22%3A%22en%22%7D)**
<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>
  
  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary> 
  
  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>
  
  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>
  
  We typically assign the developer who opened the issue in the QA repository.

  </details>
  
  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>
  
  You should:

  1. Consult the [QA Book](https://github.com/phetsims/QA/blob/master/doc/qa-book.md).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by ariel-phet** [comment]
High priority @KatieWoe but after #404

**Comment by KatieWoe** [comment]
Wave Interference Memory Test Results
![WImemory](https://user-images.githubusercontent.com/41024075/63202302-4dfa3e00-c046-11e9-8070-a4c4d8cc827e.png)

**Comment by KatieWoe** [comment]
Waves Intro memory test:
![introductionofwaves](https://user-images.githubusercontent.com/41024075/63295335-9fded600-c289-11e9-89d0-863440c770b0.png)

**Comment by KatieWoe** [comment]
Large amount of lag on 4th screen in Edge, especially the slider that controls light color. I'm guessing we're not concerned enough for an issue @ariel-phet?

**Comment by KatieWoe** [comment]
QA is done

**Comment by samreid** [comment]
All issues have been discussed, deferred or addressed.  I'll continue in https://github.com/phetsims/wave-interference/issues/441 to create the next RC.
